justspacesapp/templates/registration/signup.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h2>Sign up</h2>
+  <form method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
```

Comment:
Currently, form errors do not get logged clearly: mainly, they appear in black, within a bulleted list. It would be best if the error message and corresponding form field appears in red.

You can do this several ways. (Though...we still need to decide if we want the browser to handle validating required fields.) Can you plan to implement a solution when you work on site design, this week?

Reply:
Definitely!